shared-libs/message-utils/src/index.js

```diff
@@ -168,7 +168,7 @@ var formatDate = function(config, text, view, formatString) {
   if (!isNaN(date)) {
     date = parseInt(date, 10);
   }
-  return moment(date).format(formatString);
+  return moment.utc(date).format(formatString);
```

Comment:
This library is also used in the UI so this change means the date will be shown in UTC not in the user's timezone which is definitely a step backwards.

Reply:
const config = { date_format: 'DD-MMM-YYYY' };
console.log(moment('2016-03-06T03:45:41.000Z').format(config.date_format))
05-Mar-2016

console.log(moment.utc('2016-03-06T03:45:41.000Z').format(config.date_format))
06-Mar-2016

We need this library to format that date using utc. This fixes a couple of unit tests that you can reproduce by updating your timezone.

Reply:
I don't care about the tests. We need to support our users and our users need to see the date in their timezone. If we can't get the tests to test the correct solution, delete the tests.